src/styles/abstracts/_vars-color.scss

```diff
@@ -1,80 +1,7 @@
 /* stylelint-disable color-no-hex */
```

Comment:
We moved all hex declarations into the tokens so we can remove this

Reply:
We would need to keep the display variables since those aren't in tokens and we use them in the docs site. Do we want to add those to tokens? Or keep the display variables scoped to docs?